### PR TITLE
Cus 2462 odt soft text

### PR DESCRIPTION
**Pull request by rrobison44**
Simple change enable soft text on the initial opt-in/opt-out screen.

[Jira CUS-2462](https://banno-jha.atlassian.net/browse/CUS-2462)

### CODE REVIEW

File: odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW [hunk 1]
```diff
@@ -795,6 +829,11 @@ PROCEDURE READCONFIGFILESETTINGS
      PRINT FORMAT("OO-99:",TMPLOOP)+"'"+PARAMOPTOUTINFOTEXT(TMPLOOP)+"'"
      NEWLINE
     END
+   FOR TMPLOOP=1 TO PARAMOPTINOUTINFOTEXTLINES
+    DO
+     PRINT FORMAT("OO-99:",TMPLOOP)+"'"+PARAMOPTINOUTINFOTEXT(TMPLOOP)+"'"
```
Comment (tkainz) [inline odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW:838]: Wouldn't you want this statement to be "PRINT FORMAT("OIO-99:"........  For PARAMOPTINOUTINFOTEXT(TMPLOOP)

**Review by tkainz — COMMENTED** [review]
Just the one minor type-o

File: odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW (hunk 1 above)
Comment (rrobison44) [reply odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW]: Sure enough.  Fixed.  Thanks for catching that.

**Review by tkainz — APPROVED** [review]

**Review by jucarson — APPROVED** [review]
Looks Good.

**Review by jordan-keenan — COMMENTED** [review]
Looks good

**Pull request merged**